fearless/Modules/AssetSelection/AssetSelectionViewFactory.swift

```diff
@@ -0,0 +1,55 @@
+import Foundation
+import RobinHood
+import SoraFoundation
+
+struct AssetSelectionViewFactory {
+    static func createView(
+        delegate: AssetSelectionDelegate,
+        selectedChainId: ChainAssetId?,
+        assetFilter: @escaping AssetSelectionFilter
+    ) -> ChainSelectionViewProtocol? {
+        let repository = ChainRepositoryFactory().createRepository(
+            for: nil,
```

Comment:
`nil` here without a parameter label brings some confusion. Seem like it is better to omit it, considering that it has a default value